tests/override_config_table/test_override_config_table.py

```diff
@@ -59,7 +59,7 @@ def setup_env(duthosts, golden_config_exists_on_dut, tbinfo, enum_rand_one_per_h
     config_reload(duthost, config_source="minigraph", safe_reload=True)
     running_config = get_running_config(duthost)
 
-    yield running_config
+    yield running_config, duthost
```

Comment:
why do we need duthost?
enum_rand_one_per_hwsku_hostname is a fixture, will it return different host in the same test?